python/lsst/daf/butler/datastores/chainedDatastore.py

```diff
@@ -934,3 +947,75 @@ def export_records(self, refs: Iterable[DatasetIdRef]) -> Mapping[str, Datastore
                 all_records[name] = record_data
 
         return all_records
+
+    def export(
+        self,
+        refs: Iterable[DatasetRef],
+        *,
+        directory: Optional[ResourcePathExpression] = None,
+        transfer: Optional[str] = "auto",
```

Comment:
Why not? I think the base class was written before I came up with the "auto" meaning (when `transfer=None` means "don't transfer" and we determined that was generally a bad default but the base class doesn't really have a choice because we haven't used an enum for the transfer options.

Reply:
I was thinking that the client can call the method assuming that default for `transfer` is `None`, as declared in the base class, but in reality `transver` will be "auto" (not sure if there is a difference between the two).

Reply:
For a chained datastore `transfer=None` could be a bit tricky. `None` means "do not export any files to the given directory, leave them in place" which is fine if you have a single datastore but in a chained datastore you could end up with some entries from one datastore and also entries from another. I suppose I could switch to "auto" if `directory != None` rather than raise below in that scenario.